tests/end_to_end/misc1/irix-o2-validsyntax-out.c

```diff
@@ -1,7 +1,8 @@
 extern s32 D_410170;
 extern MIPS2C_UNK D_410178;
-extern MIPS2C_UNK func_00400140;
-extern MIPS2C_UNK func_00400158;
+extern s32 (*func_00400140)(MIPS2C_UNK, MIPS2C_UNK, s32, MIPS2C_UNK, s32);
+extern MIPS2C_UNK (*func_00400158)(MIPS2C_UNK32, s32, s32);
```

Comment:
This doesn't seem right - it should be able to tell that these are bare fns and not fn pointers. I'm going to investigate this a bit further.